src/ui/util.rs

```diff
@@ -14,3 +16,37 @@ pub fn format_code_time(time: Option<u64>) -> String {
         None => "--".to_owned(),
     }
 }
+pub fn format_test_groups(groups: &Option<Vec<u64>>) -> String {
+    match groups {
+        Some(groups) => {
+            let mut text: String = groups
+                .iter()
+                .map(|&group| group.to_string() + ",")
+                .collect();
+            text.pop(); // Removes the trailing comma
+            text
+        }
+        None => "".to_owned(),
+    }
+}
+
+pub fn styled_task_status_or_score(
+    status: Option<TaskStatus>,
+    score: Option<u64>,
+) -> StyledObject<String> {
+    if let Some(points) = score {
+        return match points {
+            points if points >= 100 => style(points.to_string()).black().on_green(),
+            points if points > 60 => style(points.to_string()).green(),
+            points if points > 10 => style(points.to_string()).yellow(),
+            _ => style(points.to_string()).red(),
+        };
+    }
+
+    match status {
+        Some(TaskStatus::Pass) => style("+".to_string()).bold().green(),
+        Some(TaskStatus::Fail) => style("X".to_string()).bold().red(),
+        Some(TaskStatus::None) => style("-".to_string()).bold().dim(),
+        None => style("-".to_string()).dim(),
```

Comment:
Intentional for the `None` case to be different?
```suggestion
        Some(TaskStatus::None) | None => style("-".to_string()).bold().dim(),
```

Reply:
We also discussed making points bold